paasta_tools/utils.py

```diff
@@ -294,6 +294,10 @@ def get_pool(self):
         :returns: the "pool" attribute in your config dict, or the string "default" if not specified."""
         return self.config_dict.get('pool', 'default')
 
+    def get_pool_constraints(self):
+        pool = self.get_pool()
+        return [["pool", "LIKE", pool]]
```

Comment:
I was aware of this, and I adjusted the words to match. I agree it is kinda a tossup and went with LIKE for consistency.